README.md

```diff
@@ -1,27 +1,36 @@
 # On-Demand Rides and Deliveries iOS Samples
 
-This repository contains source code of the following samples.
+This repository contains source code for the following sample apps.
 
-1. Driver SDK sample, available in two languages:
+1. The Driver SDK sample, available in two languages:
    - Swift: in `swift/driver_swiftui` directory.
    - Objective-C: in `objectivec_samples/Driver` directory.
-2. Consumer SDK sample, available in two languages:
+2. The Consumer SDK sample, available in two languages:
    - Swift: in `swift/consumer_swiftui` directory.
    - Objective-C: in `objectivec_samples/Consumer` directory.
 
-You can use the following guide to build and run a basic consumer and driver app integrated with the On-demand Rides and Deliveries Solution backend services. After completing this guide, you should have an app that can display an active trip, respond to trip updates, and handle trip errors.
+Use the following guide to build and run a basic consumer and a driver app
+that uses these SDK samples. This will help you integrate your app with the
+On-demand Rides and Deliveries Solution backend services. 
 
-Note: Before building your app, you need to have your provider services in place including vehicle to consumer matching functionality. Specifically, you need the Cloud Project ID unique to the service provider the consumer app supports.
+After completing this guide, you should have an app that can display an active
+trip, respond to trip updates, and handle trip errors.
 
 ## Prerequisites
-1. Please fully complete [Getting Started with Fleet Engine](https://developers.google.com/maps/documentation/transportation-logistics/on-demand-rides-deliveries-solution/trip-order-progress/fleet-engine)
-2. Please make sure the [provider backend](https://github.com/googlemaps/java-on-demand-rides-deliveries-stub-provider)
-is up and running.
-3. To access the CocoaPods of Consumer and Driver SDKs, please make sure to be a member of the
-Google group [google-maps-odrd-access-external](https://groups.google.com/a/google.com/g/google-maps-odrd-access-external).
-5. Please authenticate by visiting https://cpdc-eap.googlesource.com/new-password
-and setting up a git cookie.
-6. Install CocoaPods on the development machine
+
+This guide assumes you have done all of the following.
+
+1. Established a Cloud project service account with a project ID needed
+   for running the sample apps. 
+2. Set up Fleet Engine as described in 
+   [Getting Started with Fleet Engine](https://developers.google.com/maps/documentation/transportation-logistics/on-demand-rides-deliveries-solution/trip-order-progress/fleet-engine)
+3. Set up the [provider backend](https://github.com/googlemaps/java-on-demand-rides-deliveries-stub-provider)
+   and have it running.
+4. Joined the Google group [google-maps-odrd-access-external](https://groups.google.com/a/google.com/g/google-maps-odrd-access-external)
```

Comment:
I wasn't able to join this from my outside account. Will they know how to request access?